JRIngram/maze-gen
# Update README and CONTRIBUTING to have links to github

# Pull Request (PR) - *Add links to CONTRIBUTING and README*
## Description of Change
Update README and CONTRIBUTING to have links to github


## Link To Issue 
N/A


## How to test this change
Click all the links in the readme and contributing and check that it links to the github pages


## Checklists
### Checklist for developer of feature

### Checklist for reviewers
